wandb/sdk/wandb_run.py

```diff
@@ -671,6 +671,18 @@ def _init(
                 os.path.join("code", self._settings.program_relpath)
             )
 
+        if self._settings.fork_from is not None:
+            # verify that the specified project matches the current project.
+            # forking from a different project or entity is not currently supported.
+
+            config[wandb_key]["branch_point"] = {
+                "run_id": self._settings.fork_from.run,
+                "step": self._settings.fork_from.value,
+            }
+
+            self._step = int(self._settings.fork_from.value)
+            self._starting_step = int(self._settings.fork_from.value)
```

Comment:
i'm pretty sure if you did everything correctly, you don't need to set these

Reply:
fixed